app/views/questions/_comment.html.erb

```diff
@@ -1,13 +1,31 @@
 <div>
     <% @question.comments.each do |comment| %>
+   
+    <% if user_signed_in? && comment.question.user_id == current_user.id && 
+    comment.user_id == current_user.id %>
+    <div class="note">
+    <% end %>
+
     <h4>Comment by <%= comment.user.username%></h4>
     <p class="comment"><%= comment.comment_text %></p>
+
+    <% if user_signed_in? && comment.question.user_id == current_user.id && 
+    comment.user_id == current_user.id %>
```

Comment:
@jia-hern i think this additional condition shouldn't be there? 
if i log in as another user, i should also see the comment being colored differently.

screenshot below, on left logged in as "ruby", on the right logged in as the OP "micheline"
![Screenshot 2020-08-11 at 10 44 56 PM](https://user-images.githubusercontent.com/11924643/89911690-4acb6b00-dc24-11ea-9db9-200730783cda.png)